### PR TITLE
upaded Mapform to only save one location

### PR DESCRIPTION
**Pull request by Leepak777**
updated Mapform to only save one location instead of an array
now it should only display the latest clicked location

### CODE REVIEW

File: frontend/src/components/CreateMemo/components/LocationMapPicker/Map.tsx [hunk 1]
```diff
@@ -33,40 +31,39 @@ const MapForm = ({ selectedLocation, onMapClick }: any) => {
...
         setInitialPosition(clickedLocation);
       },
     });
 
     const deleteMarker = (index: any, e: any) => {
       e.stopPropagation(); // Stop event propagation to prevent the map click event
-      const updatedPositions = [...positions];
-      updatedPositions.splice(index, 1); // Remove the position at the specified index
-      setPositions(updatedPositions);
+      setPosition(null);
```
Comment (Toastjpg) [inline frontend/src/components/CreateMemo/components/LocationMapPicker/Map.tsx:42]: The user shouldnt be allowed to delete the marker if there is only one marker to place. 

A location marker should me mandatory for a memo.

**Comment by Toastjpg** [comment]
also update the comments since locations is no longer an array now

**Review by Toastjpg — APPROVED** [review]
lgtm otherwise

File: frontend/src/components/CreateMemo/components/LocationMapPicker/Map.tsx (hunk 1 above)
Comment (Leepak777) [reply frontend/src/components/CreateMemo/components/LocationMapPicker/Map.tsx]: removed the delete button and popup for the marker

**Review by Toastjpg — APPROVED** [review]

**Review by jerrydngzh — APPROVED** [review]
lgtm, merge when you're ready

**Pull request merged**